homework_1/task_1/fizzbuzz/fizzbuzz.go

```diff
@@ -0,0 +1,32 @@
+package fizzbuzz
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+func Fizzbuzz(start, end int) ([]string, error){
```

Comment:
Consider how this is going to be used - the call will look something like `fizzbuzz.Fizzbuzz(...)`. This is usually an anti-pattern in go, naming something within a package with the same name as the package name.

Some alternative names to consider here:
`fizzbuzz.GetForRange(...)`
`fizzbuzz.GenerateForRange(...)`
etc...